### PR TITLE
First commit

### PR DESCRIPTION
**Pull request by MannyShussan**
- Deploy and create a Pull Request with a [DEMO LINK](https://MannyShussan.github.io/layout_landing-page/)

### CODE REVIEW

**Review by joaorpereira — CHANGES_REQUESTED** [review]
1. convert all images and icons files to lowercase, this is causing issues when you type the files to be imported
2. verify all paths where you are importing the images, a lot of paths are incorrect. use ctrl + click on the path to verify if the path is valid or not

File: src/styles/style.css.map [hunk 1]
```diff
@@ -0,0 +1 @@
...CE;;AACA;EP+BA;EAEA;EACA;EACA;EOhCE;;APFF;EODA;IPsCE;IAEA;;;AAjCF;EOPA;IP4CE;;;AOtCF;EACE;EAEA;EP2CF;;AACA;EACE,OEpDc;;AFDhB;EOKA;IAQI;;;APPJ;EODA;IAYI;;;;ACnBJ;EACE;ERoDF;;AACA;EACE,WQpD0B;;;ACJ9B;EACE;EACA;EACA;EACA;;ATDA;ESGA;IAEI;IACA;IACA;;;;ACVN;EACE;EACA;EAEA;EACA;EACA;EACA;EAEA,ORNc;;AFAd;EUHF;IAYI;IAEA;;;AVLF;EUTF;IAkBI;IAEA;;;;ACpBJ;EACE;EACA,OTCc;ESCd;;AAEA;EACE;EACA;;AAGF;EACE;EACA;EACA;EACA;EAEA;EACA;EACA;EACA;;AAEA;EACE;EACA;EACA;;AAIJ;EACE;;;AC9BJ;EACE;EACA;EACA;;AAGA;EACE;EACA;EACA;EACA;;AAGF;EACE;EACA;EACA;EACA;EACA;;AZfF;EYUA;IAQI;;;AAIJ;EACE;;AZvBF;EYsBA;IAII;;;;AC7BN;EACE;EACA;EACA;EACA;;AbDA;EaHF;IAOI;;;AAGF;EACE;EACA;;AAGF;EACE;;AAGF;EACE;EbkCF;;AACA;EACE,WalC0B;;AbnB5B;EagBA;IAMI;;;;ACzBN;EACE;EACA;EACA;EACA;EACA;EACA;EdgDA;;AACA;EACE,WchDwB;;AdL1B;EcHF;IAWI;IACA;;;AAGF;EACE;;AAGF;EACE;;AAEA;EACE;EACA;;AAIJ;EACE;;;AC7BJ;EACE;EACA;EACA;EACA;EAEA;EACA;;AAEA;EACE;EACA;EACA;EACA;EAEA;;AfZF;EeMA;IASI;;;AAIJ;EACE;;AAGF;EACE;EACA;EACA;EACA;EACA;EACA;;AAGF;EACE;EACA;EAEA;EACA;EAEA;EACA;EACA;EACA;EACA;EACA;;;AC9CF;EACE;EACA;EACA;EAEA;EACA;EACA;;AAGF;EACE;EACA;EACA;EACA;EACA;;AAEA;EACE;EACA;EACA;EACA;EACA;EACA;EACA,kBdtBU;EcwBV;EACA;EACA;;AAGF;EACE","file":"style.css"}
```
Comment (joaorpereira) [inline src/styles/style.css.map:1]: why this file is generated? delete this file, letś just use scss

File: src/styles/style.css [hunk 2]
```diff
@@ -0,0 +1,765 @@
+/* stylelint-disable import-notation */
+.container {
```
Comment (joaorpereira) [inline src/styles/style.css:2]: why this file is generated? delete this file, letś just use scss

File: src/index.html [hunk 3]
```diff
@@ -1,22 +1,463 @@
...
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Manrope:wght@200..800&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
-      href="./styles/main.scss"
+      href="./styles/style.scss"
```
Comment (joaorpereira) [inline src/index.html:34]: `  href="./styles/main.scss"`

File: src/styles/style.scss [hunk 4]
```diff
@@ -0,0 +1,18 @@
+/* stylelint-disable import-notation */
```
Comment (joaorpereira) [inline src/styles/style.scss:1]: move this content to `main.scss` file an delete the file style.scss


```
@import '../utils/variables.scss';
@import '../utils/mixins.scss';
@import './blocks/contacts-block';
@import './blocks/button';
@import './blocks/form';
@import './blocks/about-us';
@import './blocks/categories';
@import './blocks/recommended';
@import './blocks/product';
@import './blocks/main';
@import './blocks/section-title';
@import './blocks/page';
@import './blocks/header';
@import './blocks/top-bar';
@import './blocks/icon';
@import './blocks/menu';
@import './blocks/nav';
```

File: src/styles/blocks/icon.scss [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+  @include hover(transform, scale(1.2));
+
+  @include on-tablet {
+    height: 32px;
+    width: 32px;
+  }
+
+  &--phone {
+    background-image: url(../images/icons/Icon-Phone-Call.svg);
```
Comment (joaorpereira) [inline src/styles/blocks/icon.scss:17]: `background-image: url(../images/icons/Icon-Phone-call.svg);`

**Comment by julianFBarbosa** [comment]
Demo link is not available
image

File: src/styles/blocks/icon.scss [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+    height: 32px;
+    width: 32px;
+  }
+
+  &--phone {
+    background-image: url(../images/icons/icon-phone-Call.svg);
+  }
+
+  &--menu {
+    background-image: url(../images/icons/icon-burger-menu.svg);
```
Comment (joaorpereira) [inline src/styles/blocks/icon.scss:21]: **Inconsistent Case in File Names:**

CSS URLs are case-sensitive. For example:
icon-burger-menu.svg is lowercase, while later you reference Icon-Burger-menu.svg with mixed case. If your actual file names don't match this case exactly, the images won't load.

https://developer.mozilla.org/en-US/docs/Web/CSS/background-image

File: src/styles/blocks/icon.scss [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+
+    &:hover {
+      transform: scale(1.2);
+      background-image: url(../images/icons/icon-burger-menu-hover.svg);
```
Comment (joaorpereira) [inline src/styles/blocks/icon.scss:25]: **Inconsistent Case in File Names:**

CSS URLs are case-sensitive. For example:
icon-burger-menu.svg is lowercase, while later you reference Icon-Burger-menu.svg with mixed case. If your actual file names don't match this case exactly, the images won't load.

https://developer.mozilla.org/en-US/docs/Web/CSS/background-image

File: src/styles/blocks/icon.scss [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+    }
+  }
+
+  &--close {
+    background-image: url(../images/icons/icon-close.svg);
```
Comment (joaorpereira) [inline src/styles/blocks/icon.scss:30]: **Inconsistent Case in File Names:**

CSS URLs are case-sensitive. For example:
icon-burger-menu.svg is lowercase, while later you reference Icon-Burger-menu.svg with mixed case. If your actual file names don't match this case exactly, the images won't load.

https://developer.mozilla.org/en-US/docs/Web/CSS/background-image

File: src/styles/blocks/icon.scss [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+  }
+}
```
Comment (joaorpereira) [inline src/styles/blocks/icon.scss:32]:
```
.icon {
  display: block;
  height: 24px;
  width: 24px;
  background-image: url("../images/icons/Icon-Burger-menu.svg");
  background-size: cover;
  transition: transform $effectDuration;

  @include hover(transform, scale(1.2));

  @include on-tablet {
    height: 32px;
    width: 32px;
  }

  &--phone {
    background-image: url("../images/icons/Icon-Phone-call.svg");
  }

  &--menu {
    background-image: url("../images/icons/Icon-Burger-menu.svg");

    &:hover {
      transform: scale(1.2);
      background-image: url("../images/icons/Icon-Burger-menu-hover.svg");
    }
  }

  &--close {
    background-image: url("../images/icons/Icon-Close.svg");
  }
}

```

**Comment by joaorpereira** [comment]
@MannyShussan please be sure to run: `npm run deploy`

to verify your code errors you can also run: `npm run deploy -- -l`

error:

78606dfd-f820-4262-878e-db5aff47ee7e (1)

**Review by raulriato — CHANGES_REQUESTED** [review]
Hey, @MannyShussan, can you confirm if you ran the command `npm run deploy`? If you do not run it, we aren't able to review your application. Make sure to run it, and if you have any troubles, call for help in the chat :)

**Comment by MannyShussan** [comment]
I remember having run command

**Comment by MannyShussan** [comment]
I ran again. Done! check again

File: src/index.html [hunk 10]
```diff
@@ -1,22 +1,463 @@
...
+              </article>
+            </div>
+          </section>
+          <section
+            class="contact-us"
+            id="contact-us"
+          >
+            <h2 class="section-title section-title--contact-us">contact-us</h2>
+
+            <form
```
Comment (julianFBarbosa) [inline src/index.html:350]: Form validation isn't working properly.


https://github.com/user-attachments/assets/834a768c-40b0-46e5-bd3e-0147f3228b9c

File: src/index.html [hunk 11]
```diff
@@ -1,22 +1,463 @@
...
+          >
+            <div class="photo-news">
+              <img
+                src="./images/about-us.png"
+                alt="news"
+                class="photo-news__img"
+              />
+            </div>
+
+            <div class="about-us__news">
```
Comment (julianFBarbosa) [inline src/index.html:331]: the `about` section has the same background as the `contact-us`, this isn't a explicit requirement on the checklist but it gives a weird appearance on the website.

image


on Figma this section looks like this:

image

File: src/index.html [hunk 12]
```diff
@@ -1,22 +1,463 @@
...
+              Your senses.
+              <br />
+              Set free.
+            </h1>
+          </div>
+        </div>
+      </div>
+    </header>
+
+    <aside
```
Comment (julianFBarbosa) [inline src/index.html:80]: This gray background does not exist on Figma, giving an strange look to the menu.

image


on Figma:

image

File: src/index.html [hunk 13]
```diff
@@ -1,22 +1,463 @@
...
+                transcends time by bridging the gap between one of our iconic
+                designs from 1972 and today's cutting-edge digital technology.
+              </article>
+            </div>
+          </section>
+          <section
+            class="contact-us"
+            id="contact-us"
+          >
+            <h2 class="section-title section-title--contact-us">contact-us</h2>
```
Comment (julianFBarbosa) [inline src/index.html:348]: I think that you forgot to change the title from `contact-us` to `Contact Us`

**Comment by MannyShussan** [comment]
I made some changes, check it out

File: src/index.html [hunk 14]
```diff
@@ -1,22 +1,446 @@
...
+                      class="category__photo"
+                    />
+                  </a>
+                </div>
+
+                <h3 class="category__title">Smart Home</h3>
+              </article>
+
+              <div class="category__button">
+                <button class="button">
```
Comment (joaorpereira) [inline src/index.html:305]: this button is not styled well

verify outline property of the button

the ui of button "send" is correct

File: src/index.html [hunk 15]
```diff
@@ -1,22 +1,446 @@
...
+        <section
+          class="recommended"
+          id="recommended"
+        >
+          <h2 class="section-title">Recommended</h2>
+
+          <div class="recommended__products">
+            <a
+              href="#"
+              class="product recommended__product"
```
Comment (joaorpereira) [inline src/index.html:163]: the correct color of text is not blue

File: src/index.html [hunk 16]
```diff
@@ -1,22 +1,446 @@
...
+              <p class="product__category">
+                Huge sound anywhere – in the home or outdoors. Beosound A5 is a
+                Wi-Fi and Bluetooth speaker with long-lasting battery and
+                wireless charging built in.
+              </p>
+              <p class="product__price">$ 1299.00</p>
+            </a>
+            <a
+              href="#"
+              class="product recommended__product"
```
Comment (joaorpereira) [inline src/index.html:180]: the correct color of text is not blue

File: src/index.html [hunk 17]
```diff
@@ -1,22 +1,446 @@
...
+              <p class="product__category">
+                Wireless noise-cancelling headphones. Listen in comfort whether
+                you're on a call, working from home or taking a moment for
+                yourself.
+              </p>
+              <p class="product__price">$ 499.00</p>
+            </a>
+            <a
+              href="#"
+              class="product recommended__product"
```
Comment (joaorpereira) [inline src/index.html:197]: the correct color of text is not blue

File: src/index.html [hunk 18]
```diff
@@ -1,22 +1,446 @@
...
+                </button>
+              </div>
+            </div>
+          </div>
+        </section>
+        <section
+          class="about-us"
+          id="about-us"
+        >
+          <div class="about-us__banner"></div>
```
Comment (joaorpereira) [inline src/index.html:321]: this banner should fill the hole width of the page. right now it respected the padding of the page. you should remove this section from the section that contain the padding logic and after this section add back this padding value

**Review by joaorpereira — CHANGES_REQUESTED** [review]

**Pull request closed without merging**